Makefile

```diff
@@ -17,6 +17,7 @@ build:
 	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
 	mvn package -DskipTests=true
 	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
+	mvn org.owasp:dependency-check-maven:check -DassemblyAnalyzerEnabled=false
```

Comment:
I don't think this line should be added. It's line number 62 that needs changing, to add the flag that causes a build failure, as per comments in lines above. I'd probably go for the `-DfailBuildOnAnyVulnerability=true` setting.

Reply:
I have removed the line and added the check after line 62

Reply:
👍 thanks. Please see new comments added below.